seabird_chat_ingest.proto

```diff
@@ -0,0 +1,144 @@
+syntax = "proto3";
```

Comment:
Are there any events that a chat backend could send to core and expect a response for? It seems like all now are either `respond to core` or `do something with no ack`.

Reply:
Anything where a chat backend would call into core would be implemented as rpc rather than messages on the stream.

Reply:
Does that mean that LeaveChannel should be an RPC instead? What happens if the bot is kicked?

Reply:
No, this is specifically for chat backends calling in to core, so if an event happens, it should be exposed via the stream. If the backend actually needs to do something, we will need to add APIs for it to call in to core.

Also, if a bot was kicked, a leave channel event will be generated

Reply:
> Anything where a chat backend would call into core would be implemented as rpc rather than messages on the stream.
> No, this is specifically for chat backends calling in to core, so if an event happens, it should be exposed via the stream.

These two messages are saying opposite things. Which is it? The first half of your response also seems to contradict the second half :(

> Also, if a bot was kicked, a leave channel event will be generated

This goes against what you said as well--why wouldn't this be an RPC?

Reply:
We resolved this offline.